packages/core-sdk/src/resources/dispute.ts

```diff
@@ -56,22 +56,24 @@ export class DisputeClient {
   public async raiseDispute(request: RaiseDisputeRequest): Promise<RaiseDisputeResponse> {
     try {
       const liveness = BigInt(request.liveness);
-      const bonds = BigInt(request.bond);
       const [minLiveness, maxLiveness] = await Promise.all([
         this.arbitrationPolicyUmaClient.minLiveness(),
         this.arbitrationPolicyUmaClient.maxLiveness(),
       ]);
 
+      const [minimumBond, maxBonds] = await Promise.all([
```

Comment:
minor, but we should do this after the liveness validation. we don't need 2 rpc queries to fetch bonds if other validation will fail.
In terms of priority, 1st validate all inputs that don't require rpc calls, then validate each input that require rpc call one after another.